src/cpu/lowrisc/ibex/src/lib.rs

```diff
@@ -4,3 +4,9 @@
 pub fn init() {
     arch::init()
 }
+
+// There is no ibex-specific way of halting yet.
+
+pub fn halt() -> ! {
```

Comment:
Could you instead do:

```
pub use arch::halt
```

I think something like this is possible in rust?

Reply:
yes. done.